spec/support/models.rb

```diff
@@ -0,0 +1,9 @@
+module AbcCompany
+  class SomethingApprovable
+  end
+end
+
+module SomeCompany
+  class SomethingApprovable
+  end
+end
```

Comment:
Tried removing them – they're required because of https://github.com/18F/C2/blob/4a1b9c180aafd96ab1966c85808514d0175e9e18/app/policies/proposal_policy.rb#L113.